src/lazy-loading/LazyRelationsWrapper.ts

```diff
@@ -105,9 +106,9 @@ export class LazyRelationsWrapper {
             .from(relation.type, relation.propertyName) // Category, category
             .innerJoin(relation.entityMetadata.target as Function, relation.entityMetadata.name, conditions);
 
-        joinColumns.forEach(joinColumn => {
-            qb.andWhere(`${relation.entityMetadata.name}.${joinColumn.referencedColumn!.databaseName} = :${joinColumn.referencedColumn!.databaseName}`)
-                .setParameter(`${joinColumn.referencedColumn!.databaseName}`, joinColumn.referencedColumn!.getEntityValue(entity));
+        primaryColumns.forEach(primaryColumn => {
+            qb.andWhere(`${qb.escape(relation.entityMetadata.name)}.${qb.escape(primaryColumn.databaseName)} = :${primaryColumn.databaseName}`)
+                .setParameter(`${primaryColumn.databaseName}`, primaryColumn.getEntityValue(entity));
```

Comment:
Can you replace:

```ts
qb.andWhere(`${qb.escape(relation.entityMetadata.name)}.${qb.escape(primaryColumn.databaseName)} = :${primaryColumn.databaseName}`)
    .setParameter(`${primaryColumn.databaseName}`, primaryColumn.getEntityValue(entity));
```

to

```ts

qb.andWhere(`${relation.entityMetadata.name}.${primaryColumn.propertyPath} = :${primaryColumn.propertyName}`)          
.setParameter(`${primaryColumn.propertyName}`, primaryColumn.getEntityValue(entity));
```

It must give same result as right now and automatically escape column names